electron/app/components/DisplayOptionsSidebar.tsx

```diff
@@ -14,6 +15,8 @@ import CheckboxGrid from "./CheckboxGrid";
 import DropdownCell from "./DropdownCell";
 import SelectionTag from "./Tags/SelectionTag";
 import { Button } from "./utils";
+import { colorMap as colorMapAtom } from "../recoil/atoms";
+import { refreshColorMap as refreshColorMapSelector } from "../recoil/selectors";
```

Comment:
I would suggest the `import * as atoms` approach used elsewhere for consistency - names would contain the same information (`atoms.colorMap` instead of `colorMapAtom`). Is there a benefit to doing it this way?

Reply:
I am not particularly opinionated about this in the short-term. I will need to learn more about bundling, tree-shaking, etc. In short, I imported in the above manner to note that this is an important topic that doesn't need to be solved today.

https://stackoverflow.com/questions/40061042/import-vs-import-specificname-in-typescript-es6

Taking a step back, I do think that throwing every atom into `atoms` and every selector into `selectors` is not the path forward, and then importing `* as ...` everywhere. I don't think what I've done here is great either, though.

I'll change to `as atoms` for consistency.

Reply:
I think most of the discussion on this subject is related to importing functions - importing specific functions allows the bundler to strip out functions that aren't used at all, while importing the entire module prevents that (in most cases) since functions could be accessed dynamically. It doesn't apply as directly to this situation because we're importing objects, which always exist even if they're unused (partly because their creation has side effects).

I think importing specific objects here would trigger a warning if they don't exist, which would definitely be helpful. Some disadvantages (why I went with `import *` originally) are that objects frequently have to be renamed, since they tend to share names with their corresponding values within components, and sometimes a lot of objects are needed by a single component.

Definitely worth discussing later - I just thought I'd point it out here.